examples/event.js

```diff
@@ -0,0 +1,144 @@
+// event.js
+global.WebSocket = require('ws');
+const Client = require('../client');
+// NOTE: Event support is only on API version 4.0.0+
+
+const { DBMessaging } = require('../generated/containerPb');
+
+/**
+ * Helper: Create an exact match condition for filtering.
+ * Uses the EventQuery.Condition message.
+ */
+function createExactCondition(value) {
+  return DBMessaging.Protobuf.EventQuery.Condition.create({
+    value: value,
+    type: DBMessaging.Protobuf.EventQuery.MatchType.Exact // 0
+  });
+}
+
+/**
+ * Helper: Create a ConditionList from an array of conditions.
+ */
+function createConditionList(conditions) {
+  return DBMessaging.Protobuf.EventQuery.ConditionList.create({
+    conditions: conditions
+  });
+}
+
+/**
+ * Helper: Create sender conditions.
+ * @param {Array<string>} senders - Array of exact-match sender names.
+ * @param {Array<string>} senderPatterns - Array of wildcard patterns.
+ * @returns {DBMessaging.Protobuf.EventQuery.ConditionList}
+ */
+function createSenderConditions(senders = [], senderPatterns = []) {
+  const conditions = [
+    ...senders.map(sender => createExactCondition(sender)),
+    ...senderPatterns.map(pattern =>
+      DBMessaging.Protobuf.EventQuery.Condition.create({
+        value: pattern,
+        type: DBMessaging.Protobuf.EventQuery.MatchType.Wildcard // 1
+      })
+    )
+  ];
+  return createConditionList(conditions);
+}
+
+/**
+ * Helper: Create data conditions.
+ * @param {Object} dataConditions - Map of data field keys to values (or arrays of values)
+ * @returns {Object} - Map of field name to ConditionList
+ */
+function createDataConditions(dataConditions = {}) {
```

Comment:
Why does this code look so much more complicated than the example described in client.js?

In any case, I would prefer if end-user code would not have to directly access things from Protobuf namespace if possible, e.g. the DBMessaging.Protobuf.EventQuery.Condition.create.

![image](https://github.com/user-attachments/assets/f23cca4b-10ef-49df-83cc-a1a76609a9ed)